README.md

```diff
@@ -171,6 +171,18 @@ Dexter.withActivity(activity)
 	.check();
 ```
 
+### Force
+For some reason, you may want to force permission asking even if Android system is 
+telling that permission is granted. 
+
+```java
+Dexter.withActivity(activity)
+	.withPermission(permission)
+	.withListener(listener)
+	.force()
```

Comment:
How this affect when you are asking for several permissions instead of just a single one?